test/utils/settings-helper.js

```diff
@@ -21,7 +23,11 @@ const createSettingsInstance = async (
     Settings,
     params,
 ) => {
-    const instance = await Settings.new();
+    const { from } = Settings.class_defaults
```

Comment:
What is class_defaults?

Reply:
Its an object that is set by Truffle that holds which address should be the `from` and the `gasLimit` of the transaction when when you do `await Settings.new()`. Since we don't have access to the `accounts` object like in the test files, this was the easiest way to grab a valid account address.